app/ghostery-browser-hub/Views/OnboardingViews/Step1_CreateAccountView/Step1_CreateAccountView.jsx

```diff
@@ -58,24 +57,8 @@ const Step1_CreateAccountView = (props) => {
 	const { setSetupStep } = actions;
 	const email = user && user.email;
 
-	const [expanded, setExpanded] = useState(false);
 	const [view, setView] = useState(CREATE_ACCOUNT);
 
-	const arrowClassNames = ClassNames('Step1_CreateAccountView__arrow', {
-		up: expanded,
-		down: !expanded,
-	});
-
-	const faqRef = useRef(null);
-	const scrollToFAQ = () => {
-		faqRef.current.scrollIntoView({ behavior: 'smooth' });
-	};
-
-	const handleFAQLearnMoreClick = () => {
-		setTimeout(scrollToFAQ, 1);
-		setExpanded(!expanded);
-	};
-
```

Comment:
I THINK we are only cutting the bit of copy above "Learn More" - not the click-arrow-to-expand mechanism. If that's so, we'll need to restore this logic and the other related bits

Reply:
Double checked with Teresa, we are cutting the arrow as well